server/server.js

```diff
@@ -0,0 +1,57 @@
+import express from "express";
+import axios from "axios";
+import dotenv from "dotenv";
+import cors from "cors";
+
+dotenv.config();
+
+const app = express();
+const PORT = 8080;
```

Comment:
So rather than just have `const PORT = 8080`, I should do `const port = process.env.PORT;` in server.js and have `PORT = 8080` in my .env file? I thought you meant this as an optional thing before because this is a more simple project.

Reply:
No I think we still need to put it in the .env file because the port in production may be different.

Reply:
We definitely do! I need all the real world experience I can get.